sdk/highlight-next/package.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "@highlight-run/next",
-	"version": "4.4.1",
+	"version": "4.4.2",
 	"description": "Client for interfacing with Highlight in next.js",
 	"files": [
 		"dist",
-		"next-client.d.ts",
+		"client.d.ts",
 		"server.d.ts",
-		"client.d.ts"
+		"types.d.ts"
```

Comment:
i don't see a `client.d.ts`, only `next-client.d.ts` - is that what you meant?
```bash
 ~/w/highlight   vadim/fix-error-session-filter   ls sdk/highlight-next/dist/*.d.ts
sdk/highlight-next/dist/next-client.d.ts                   sdk/highlight-next/dist/server.edge.d.ts
sdk/highlight-next/dist/server.d.ts                        sdk/highlight-next/dist/with-highlight-edge-23c1ab0e.d.ts
```

Reply:
No. I have a `client.d.ts` file that references `dist/next-client.d.ts`. For some reason I can't have a root file named `client.ts`. It was throwing weird errors. So I prefixed the actual file with `next-` and then aliased to it in `package.json#exports`. Of course, I can have a `client.d.ts` file 🤷

Reply:
ah that's my bad, forgot the path here
```bash
 ~/w/highlight   vadim/fix-error-session-filter   ls sdk/highlight-next/client.d.ts
sdk/highlight-next/client.d.ts
```